libs/shared-web/src/hooks/useAuthController.ts

```diff
@@ -18,56 +26,61 @@ function getOrganizations() {
 }
 
 export function applyToken(token: string | null) {
-  if (token) {
-    localStorage.setItem('auth_token', token);
-    axios.defaults.headers.common.Authorization = `Bearer ${token}`;
```

Comment:
This relied on an `axios` singleton and can be problematic under some circumstances. We don't need to mutate the axios module, instead we are already fetching from `localStorage` in [api.client.ts](https://github.com/novuhq/novu/blob/refactor-auth-controller/libs/shared-web/src/api/api.client.ts#L14) for each request.